1432106208947	Sync.ErrorHandler	DEBUG	Flushing file log.
1432106208948	Sync.Service	DEBUG	Exception: Error: NS_ERROR_UNKNOWN_HOST (resource://services-sync/resource.js:402:18) JS Stack trace: waitForSyncCallback@async.js:102:6 < Res__request@resource.js:398:13 < Res_get@resource.js:422:11 < Sync11Service.prototype._fetchInfo@service.js:540:13 < sync@enginesync.js:77:15 < onNotify@service.js:1278:6 < WrappedNotify@util.js:148:20 < WrappedLock@util.js:103:15 < _lockedSync@service.js:1268:11 < sync/<@service.js:1260:13 < WrappedCatch@util.js:77:15 < sync@service.js:1248:4 < Res_get@resource.js:422:12 < Sync11Service.prototype._fetchInfo@service.js:540:14 < sync@enginesync.js:77:16 < onNotify@service.js:1278:7 < WrappedNotify@util.js:148:21 < WrappedLock@util.js:103:16 < _lockedSync@service.js:1268:12 < sync/<@service.js:1260:14 < WrappedCatch@util.js:77:16 < sync@service.js:1248:5
1432106208949	Sync.BrowserIDManager	INFO	currentAuthState returning error.login.reason.network due to previous failure
1432106208949	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1432106208949	Sync.Status	DEBUG	Status.service: error.sync.failed => error.login.failed
1432106208990	Sync.ErrorHandler	DEBUG	Log cleanup threshold time: 1431242208990
1432106208992	Sync.ErrorHandler	DEBUG	No logs to clean up.
1432106208993	Sync.ErrorHandler	DEBUG	Log cleanup threshold time: 1431242208993
1432106208994	Sync.ErrorHandler	DEBUG	No logs to clean up.
1432109808948	Sync.Service	DEBUG	User-Agent: Firefox/37.0.2 FxSync/1.39.0.20150417180058.
1432109808948	Sync.Service	INFO	Starting sync at 2015-05-20 04:16:48
1432109808952	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1432109808954	Sync.Status	INFO	Resetting Status.
1432109808954	Sync.Status	DEBUG	Status.service: error.login.failed => success.status_ok
1432109808956	FirefoxAccounts	DEBUG	already verified
1432109808957	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1432109808957	FirefoxAccounts	DEBUG	enter getAssertion()
1432109808964	FirefoxAccounts	DEBUG	getKeyPair: already have a keyPair
1432109808965	FirefoxAccounts	DEBUG	 getCertificate already had one
1432109808965	FirefoxAccounts	DEBUG	getAssertionFromCert
1432109808967	FirefoxAccounts	DEBUG	getAssertionFromCert returning signed: true
1432109808967	Sync.BrowserIDManager	DEBUG	Getting a token
1432109808968	Common.TokenServerClient	DEBUG	Beginning BID assertion exchange: https://token.services.mozilla.com/1.0/sync/1.5
1432109808972	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}}) (resource://gre/modules/services-common/tokenserverclient.js:36:35) JS Stack trace: @tokenserverclient.js:36:36 < @browserid_identity.js:14:1 < @status.js:15:1 < SyncServiceObserver.init@sync.js:65:59 < @sync.js:448:1 < require@bootstrap.js:141:4 < @main.js:18:1 < require@bootstrap.js:141:4 < startup@bootstrap.js:28:2 < XPI_callBootstrapMethod@XPIProvider.jsm:4442:9 < XPI_startup@XPIProvider.jsm:2161:13 < callProvider@AddonManager.jsm:208:12 < _startProvider@AddonManager.jsm:667:5 < AMI_startup@AddonManager.jsm:824:9 < AMP_startup@AddonManager.jsm:2402:5 < AMC_observe@addonManager.js:55:7
1432109808973	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1432109808973	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1432109808973	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1432109808973	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
